wcc-contentful/lib/wcc/contentful/action_view_rich_text_renderer.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'action_view'
+
+# An implementation of the RichTextRenderer that uses ActionView helpers to implement content_tag and concat.
+class WCC::Contentful::ActionViewRichTextRenderer < WCC::Contentful::RichTextRenderer
+  include ActionView::Helpers::TagHelper
+  include ActionView::Helpers::TextHelper
+  include ActionView::Context
+
+  # TODO: use ActionView view context to render ERB templates for embedded entries?
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/watermarkchurch/wcc-contentful/blob/814d9aace748f0276f433814fefe4b3007b1c193/wcc-contentful/lib/wcc/contentful/action_view_rich_text_renderer.rb#L11"></span>TODO added in <a href="https://github.com/watermarkchurch/wcc-contentful/blob/814d9aace748f0276f433814fefe4b3007b1c193/wcc-contentful/lib/wcc/contentful/action_view_rich_text_renderer.rb">wcc-contentful/lib/wcc/contentful/action_view_rich_text_renderer.rb</a> - is there a card associated with that?</td>
    </tr>
  </tbody>
</table>